app/scripts/store.js

```diff
@@ -358,6 +357,16 @@ var exports = (function() {
       });
     },
 
+    colorClass: function () {
```

Comment:
I would suggest to keep colors out of the store.  Put these in the component that displays it (`model-card` I believe).